.github/resources/local.repos

```diff
@@ -6,6 +6,7 @@ definitions:
 repositories:
   ros2/rosidl_typesupport_connext/COLCON_IGNORE: *empty_repo
   ros2/rmw_connext/COLCON_IGNORE: *empty_repo
```

Comment:
While you are in here, you may as well remove the `rmw_connext` line; we'll no longer check it out by default.

Reply:
oh, right, removed in 7f7784bdff6be1ceccec2291b7b39bd8b6050381

Reply:
I also removed the line for `rosidl_typesupport_connext` since that was also removed from the `ros2.repos` file: 0197ba6a285c0dfca3d29fcfa5baee1bf4c723a7